.github/CODEOWNERS

```diff
@@ -132,12 +132,7 @@
 
 # PHP
 /lib                                                                 @timothybjacobs @spacedmonkey
-/lib/global-styles.php                                               @timothybjacobs @spacedmonkey @oandregal
```

Comment:
I don't think we need to list files that were backported to WordPress 5.9.

Reply:
In this case, it has been particularly useful to have these automatic notifications to share that we shouldn't touch those files with those unaware of the process. I'd like to keep them if that's possible.

Reply:
(I'm talking the `theme.json` stuff: classes, json files)

Reply:
Sure,it is. Reverting :)